pkg/controllers/securitypolicy_controller.go

```diff
@@ -229,7 +250,11 @@ func (r *SecurityPolicyReconciler) GarbageCollector(cancel chan bool, timeout ti
 				continue
 			}
 			log.V(1).Info("GC collected SecurityPolicy CR", "UID", elem)
-			r.Service.DeleteSecurityPolicy(types.UID(elem))
+			r.counter_inc(metrics.ControllerDeleteTotal)
+			err = r.Service.DeleteSecurityPolicy(types.UID(elem))
+			if err != nil {
+				r.counter_inc(metrics.ControllerDeleteFailTotal)
+			}
```

Comment:
Do we need to r.counter_inc(metrics.ControllerDeleteSuccessTotal) if delete successfully?

Reply:
fixed, thanks.